com.unity.visualeffectgraph/Editor/GraphView/Views/DropDownButtonBase.cs

```diff
@@ -0,0 +1,218 @@
+using System;
```

Comment:
This is a reusable class to add dropdowns (a button and an arrow button to open a popup).
It's used three times in this PR